lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -695,6 +695,15 @@ void LogUnimplementedTypeKind(const char *function, CompilerType type) {
 #endif
 }
 
+CompilerType GetCanonicalClangType(CompilerType type) {
```

Comment:
Why is this called CanonicalClangType? There doesn't seem to be anything specific about clang in this function.

Reply:
I guess this would only be a canonical type if it's a clang type, since untypedefing wouldn't necessarily result in a canonical Swift type, right? Maybe add an assert or early return if this isn't a clang type?